packages/itwin/geo-tools/package.json

```diff
@@ -43,25 +43,25 @@
     "react-autosuggest": "^10.1.0"
   },
   "devDependencies": {
-    "@itwin/appui-abstract": "^3.7.1",
+    "@itwin/appui-abstract": "^4.0.0-dev.95",
```

Comment:
please use the rc, so `^4.0.0-dev.104` not 95

Reply:
should look better now